packages/onboarding/src/utils/flows.ts

```diff
@@ -12,8 +12,6 @@ export const LINK_IN_BIO_POST_SETUP_FLOW = 'link-in-bio-post-setup';
 export const CONNECT_DOMAIN_FLOW = 'connect-domain';
 export const VIDEOPRESS_FLOW = 'videopress';
 export const VIDEOPRESS_ACCOUNT = 'videopress-account';
```

Comment:
Can't we remove these flows?

Reply:
Yes, I'll remove them in a separate PR so there's less changes to review.